l10n_br_account_payment_mode/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Odoo Brazil Account Banking Payment Infrastructure',
+    'summary': 'Odoo Brazil Payments Mode',
+    'version': '8.0.1.0.0',
+    'category': 'Banking addons',
+    'license': 'AGPL-3',
+    'author': "KMEE, Odoo Community Association (OCA)",
+    "website": "https://odoo-community.org/",
+    'depends': [
+        'l10n_br_data_base',
+        'l10n_br_account_product',
+        'account_due_list_payment_mode',
+        'account_banking_payment_export',
+        'l10n_br_account_banking_payment',
+    ],
```

Comment:
these dependencies look fine but require OCA/bank-payment and OCA/account-payment to be added in the oca_dependencies.txt file of the repo.
Also this PR depends on the merge of l10n_br_account_banking_payment in this repo. I tried to extract it with its relative commits but didn't succeed yet. I'll try it again soon.